statusboard/src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,23 @@
+.App-footer {
+  width: 100%;
+  min-height: 5rem;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
```

Comment:
These don't seem to have an effect. Maybe worth thinking about removing them.

Reply:
Also, I tend to prefer that the space an element takes up be determined by padding, etc. Using height can be hard to debug for a new person later.